### PR TITLE
update upgrade tests to test from v1.14.0 to main

### PR DESCRIPTION
**Pull request by aanm**

### CODE REVIEW

**Review by joestringer — APPROVED** [review]

**Review by christarazi — APPROVED** [review]

**Comment by pchaigno** [comment]
Any specific reason why I was requested a review?

File: test/k8s/updates.go [hunk 1]
```diff
@@ -243,13 +243,6 @@ func InstallAndValidateCiliumUpgrades(kubectl *helpers.Kubectl, oldHelmChartVers
 			"hubble.relay.image.repository":          "quay.io/cilium/hubble-relay-ci",
 			"clustermesh.apiserver.image.repository": "quay.io/cilium/clustermesh-apiserver-ci",
 		}
-		if helpers.RunsOn419OrLaterKernel() {
```
Comment (aanm) [inline test/k8s/updates.go:246]: @pchaigno because you added this. I wanted to make sure there's nothing left.

**Pull request merged**